docs/conf.py

```diff
@@ -21,6 +21,8 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #sys.path.insert(0, os.path.abspath('.'))
 
+sys.path.insert(0, os.path.abspath('../src'))
```

Comment:
I've had to use this in the past to get documentation builds to properly run.  I don't think it's needed for building in `readthedocs.org`.

Reply:
Yeah. I always have to add it in too. I'll leave it and hopefully it'll make building locally easier.